### PR TITLE
lightning: add max-err summary output example

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 1]
```diff
@@ -70,7 +70,21 @@ TiDB Lightning 只能检测数据源的重复项，不能解决运行 TiDB Light
...
+| - | --- | --- | ------ |
+| 1 | Data Type | 1000 | `lightning_task_info`.`type_error_v1` |
+
+在 TiDB Lightning 的 log 文件的结尾，也会输出如下格式的错误统计信息：
+
+```shell
+[2022/03/13 05:33:57.736 +08:00] [WARN] [errormanager.go:459] ["Detect 1000 data type errors in total, please refer to table `lightning_task_info`.`type_error_v1` for more details"]
+```
+
+所有错误都会写入下游 TiDB 集群 `lightning_task_info` 数据库中的表中。在导入完成后，如果收集到报错的数据，你可以根据数据库中记录的内容，手动进行处理。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-error-resolution.md:87]:
```
<!--   **syntax_error_v1** 记录文件中的语法错误。目前尚未生效。-->
```
syntax error v1 （见 L134）目前仍未生效吗？是否可以从 L100 的示例中移除掉？

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 2]
```diff
@@ -70,7 +70,21 @@ TiDB Lightning 只能检测数据源的重复项，不能解决运行 TiDB Light
...
+
+| # | ERROR TYPE | ERROR COUNT | ERROR DATA TABLE |
+| - | --- | --- | ------ |
+| 1 | Data Type | 1000 | `lightning_task_info`.`type_error_v1` |
+
+在 TiDB Lightning 的 log 文件的结尾，也会输出如下格式的错误统计信息：
+
+```shell
+[2022/03/13 05:33:57.736 +08:00] [WARN] [errormanager.go:459] ["Detect 1000 data type errors in total, please refer to table `lightning_task_info`.`type_error_v1` for more details"]
+```
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-error-resolution.md:85]:
```suggestion
* 输出在终端的报错统计如下表所示：

    | # | ERROR TYPE | ERROR COUNT | ERROR DATA TABLE |
    | - | --- | --- | ------ |
    | 1 | Data Type | 1000 | `lightning_task_info`.`type_error_v1` |

* 输出在 TiDB Lightning 的 log 文件的结尾如下：

    ```shell
    [2022/03/13 05:33:57.736 +08:00] [WARN] [errormanager.go:459] ["Detect 1000 data type errors in total, please refer to table `lightning_task_info`.`type_error_v1` for more details"]
    ```
```

File: tidb-lightning/tidb-lightning-error-resolution.md (hunk 1 above)
Comment (glorv) [reply tidb-lightning/tidb-lightning-error-resolution.md:87]: syntax error 确实还没实现..  /cc @kennytm @gozssky  Do we have a plan to implement it?

Comment (gozssky) [reply tidb-lightning/tidb-lightning-error-resolution.md:87]: Not yet.

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by gozssky — APPROVED** [review]

**Pull request merged**